streams/src/test/java/org/apache/kafka/streams/integration/AbstractJoinIntegrationTest.java

```diff
@@ -188,6 +195,44 @@ void runTest(final List<List<KeyValueTimestamp<Long, String>>> expectedResult) t
     }
 
 
+    void runTestWithDriver(final List<List<TestRecord<Long, String>>> expectedResult) {
+        runTestWithDriver(expectedResult, null);
+    }
+
+    void runTestWithDriver(final List<List<TestRecord<Long, String>>> expectedResult, final String storeName) {
+        try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(STREAMS_CONFIG), STREAMS_CONFIG)) {
```

Comment:
The idea here is to use the existing test logic and simply use the TTD vs. the embedded broker.  No test logic was changed.